guide/src/class.md

```diff
@@ -606,9 +606,24 @@ Python::with_gil(|py| {
 
 Note that unlike class variables defined in Python code, class attributes defined in Rust cannot
 be mutated at all:
-```rust,ignore
-// Would raise a `TypeError: can't set attributes of built-in/extension type 'MyClass'`
-pyo3::py_run!(py, my_class, "my_class.my_attribute = 'foo'")
+```rust,should_panic
+# use pyo3::prelude::*;
+# #[pyclass]
+# struct MyClass {}
+# #[pymethods]
+# impl MyClass {
+#     #[classattr]
+#     fn my_attribute() -> String {
+#         "hello".to_string()
+#     }
+# }
+# 
+Python::with_gil(|py| {
+    let my_class = py.get_type::<MyClass>();
+
+    // Would raise a `TypeError: can't set attributes of built-in/extension type 'MyClass'`
+    pyo3::py_run!(py, my_class, "my_class.my_attribute = 'foo'")
+});
```

Comment:
As per #2349 shall we just change the documentation here? I'm not sure if it's worth us investing lots of effort to change this, or how hard it would be.

Reply:
I've deleted it for now.